src/components/landing-components/carousel.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import "styles/carousel.css";
+
+import carousel1 from "assets/images/carousel/book-sale-1.png";
```

Comment:
You can keep all these images in one file and you can import these files in a single line